### DISCUSSION TITLE
GSoC’24 Proposal Review | Platform parameter developer dashboard and improvements to platform parameter testing support (Saptak Manna)

### DISCUSSION THREAD

**Discussion by theMr17** [oppia/oppia-android#5351] [Proposal reviews (4. Android projects)]
Hi, 

I have drafted a proposal for the project idea [Platform parameter developer dashboard and improvements to platform parameter testing support](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2024#43-platform-parameter-developer-dashboard-and-improvements-to-platform-parameter-testing-support). Please take a look at the doc, up to the first part of section “3.2: HOW” and let me know if you have any comments. 

Proposal: [Google Doc Link](https://docs.google.com/document/d/14eeGyQnT-RGLZZi0_is6DKEwfl6LoYv4IgoOjby4NK8/edit?usp=sharing) (I have added [gsoc-2024-mentors@oppia.org](mailto:gsoc-2024-mentors@oppia.org) as an editor)

Thanks, and looking forward to your feedback.

**Comment by BenHenning** [comment]
Closing per our email discussion--this project is no longer one of the recommended for GSoC and you indicated that you'd prefer to write a proposal for the multiple classrooms (4.2) project, instead.
